includes/class-amp-story-post-type.php

```diff
@@ -1519,13 +1571,105 @@ public static function insert_attachment( $object, $cropped ) {
 		return $attachment_id;
 	}
 
+	/**
+	 * For amp_story embeds, removes the title from above the <iframe>.
+	 *
+	 * @param string  $output The output to filter.
+	 * @param WP_Post $post The post for the embed.
+	 * @return string $output The filtered output.
+	 */
+	public static function remove_title_from_embed( $output, $post ) {
+		if ( self::POST_TYPE_SLUG !== get_post_type( $post ) ) {
+			return $output;
+		}
+
+		return preg_replace( '/<blockquote class="wp-embedded-content">.*?<\/blockquote>/', '', $output );
+	}
+
+	/**
+	 * Changes the height of the AMP Story embed <iframe>.
+	 *
+	 * In the block editor, this embed typically appears in an <iframe>, though on the front-end it's not in an <iframe>.
+	 * The height of the <iframe> isn't enough to display the full story, so this increases it.
+	 *
+	 * @param string  $output The embed output.
+	 * @param WP_Post $post The post for the embed.
+	 * @return string The filtered embed output.
+	 */
+	public static function change_embed_iframe_attributes( $output, $post ) {
+		if ( self::POST_TYPE_SLUG !== get_post_type( $post ) ) {
+			return $output;
+		}
+
+		// Add 4px more height, as the <iframe> needs that to display the full image.
+		$new_height = strval( ( self::STORY_LARGE_IMAGE_DIMENSION / 2 ) + 4 );
+		return preg_replace(
+			'/(<iframe sandbox="allow-scripts"[^>]*\sheight=")(\w+)("[^>]*>)/',
+			sprintf( '${1}%s${3}', $new_height ),
+			$output
+		);
+	}
+
+	/**
+	 * Adds a new max image size to the image sizes available.
+	 *
+	 * In the AMP story editor, when selecting Background Media,
+	 * it will use this custom image size.
+	 * This filter will also make it available in the Image block's 'Image Size' <select> element.
+	 *
+	 * @param array $image_sizes {
+	 *     An associative array of image sizes.
+	 *
+	 *     @type string $slug Image size slug, like 'medium'.
+	 *     @type string $name Image size name, like 'Medium'.
+	 * }
+	 * @return array $image_sizes The filtered image sizes.
+	 */
+	public static function add_new_max_image_size( $image_sizes ) {
+		$full_size_name = __( 'AMP Story Max Size', 'amp' );
+
+		if ( isset( $_POST['action'] ) && ( 'query-attachments' === $_POST['action'] || 'upload-attachment' === $_POST['action'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Missing
+			$image_sizes[ self::MAX_IMAGE_SIZE_SLUG ] = $full_size_name;
+		} elseif ( get_post_type() && self::POST_TYPE_SLUG === get_post_type() ) {
+			$image_sizes[ self::MAX_IMAGE_SIZE_SLUG ] = $full_size_name;
+			unset( $image_sizes['full'] );
+		}
+
+		return $image_sizes;
+	}
+
```

Comment:
Not sure why it's adding the three methods above but will look into what's happening in the morning.

Reply:
Never mind, it's moved it from lower in the code. Disregard the last message.